explorer/update.go

```diff
@@ -467,7 +470,10 @@ func revertChainUpdate(tx UpdateTx, cru chain.RevertUpdate, revertedIndex types.
 	}
 	state.Metrics.Index = revertedIndex
 
-	return tx.RevertIndex(state)
+	if err := tx.RevertIndex(state); err != nil {
```

Comment:
Should be fine to punt this up the stack since there's only one possible return site

Reply:
@chris124567 doesn't look like this was changed.

Reply:
Should be fixed now